tests/v2/bad/personname-with-givenname.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="geekodoc-v2-flat.rnc" type="application/relax-ng-compact-syntax"?>
+<article xmlns="http://docbook.org/ns/docbook" version="5.2">
+  <title>Bad names</title>
+  <info>
+    <author>
+      <personname>
+        <firstname>Tux</firstname>
+        <givenname>Tix</givenname>
+        <surname>Penguin</surname>
+      </personname>
+    </author>
+    <author>
+      <personname>
+        <givenname>Geeko</givenname>
+        <givenname>Gecko</givenname>
+        <surname>Chamaeleon</surname>
+      </personname>
```

Comment:
I don't understand this -- "Geeko Gecko Chamaeleon" (given/given/sur) is a test case for a _bad name_, but "Geeko Gecko Chamaeleon jr." (given/given/sur/lineage) is a test case for a _good name_. Sound dodgy. :)

Reply:
Good catch. Actually one of the `givenname` should be `firstname`.